spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/CommonBinderProperties.java

```diff
@@ -22,7 +22,7 @@
  *
  * @author Gary Russell
  */
-public interface BinderProperties {
+public interface CommonBinderProperties {
```

Comment:
From other side any variable in the `interface` is exactly `public static final`.
I mean that we don't need those modification there.

I don't remember the discussion, but decided to stay away from interfaces for the constants containers in the SI. e.g. `FileHeaders`, `AmqpHeaders` etc.

I would agree with @markfisher regarding `BinderPropertyKeys` name...